awx/main/dispatch/queues.py

```diff
@@ -0,0 +1,42 @@
+import os
+
+from django.conf import settings
+
+
+REDIS_GROUP = "tower_settings_change"
+RSYSLOG_GROUP = "rsyslog_configurer"
+TASK_GROUP = "tower_broadcast_all"
+HEARTBEET = "web_heartbeet"
```

Comment:
These constants aren't faithfully used yet. Like the heartbeet should get applied:

https://github.com/ansible/awx/blob/301f3f25386d8b1c63d9fadcf54ab5fac9b04d1f/awx/main/management/commands/run_heartbeet.py#L64

and

https://github.com/ansible/awx/blob/fa05f555123b968f36cfd38e5a5f37d6ed786b7d/awx/main/wsrelay.py#L214